### PR TITLE
[1.x] Minor improvements + bump up to 1.x

### PR DESCRIPTION
**Pull request by clemblanco**
From #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +41,7 @@ php artisan vendor:publish --provider="PodPoint\MailExport\MailExportServiceProv
 
 You will be able to specify:
 
-* `enabled`: wether this package is enabled or not.
+* `enabled`: wether this package is enabled or not. Once installed, it's enabled by default but the `MAIL_EXPORT` environment variable can be used to configure this.
 * `disk`: which disk to use by default. `null` will use the default disk from your application filesystem.
 * `path`: the default path you would like to export your mails within a storage disk.
```
Comment (oaklees) [inline README.md:46]:
```suggestion
* `path`: the default path, within the configured disk, where mail will be exported.
```

**Review by oaklees — APPROVED** [review]
Just a couple of comments 👍🏻

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+.idea/
```
[the remark this answers is not in the document]
Comment (clemblanco) [reply .gitignore:1]: https://github.com/laravel/laravel/blob/74dfb6cec4d2c1f6d7e1a1e29e4bc9d610dc7031/.gitignore#L15-L16

🤷

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,41 @@
 
 All notable changes to `laravel-mail-export` will be documented in this file.
 
+## 2.0.0 - 2022-08-04
+
+- Laravel 9 Support **Breaking change** [#17](https://github.com/Pod-Point/laravel-mail-export/pull/17)
+- Moved from `swiftmailer/swiftmailer` to `symfony/mailer`
+- Thank you to [fridzema](https://github.com/fridzema) for contributing [#14](https://github.com/Pod-Point/laravel-mail-export/pull/14)
+
```
[the remark this answers is not in the document]
Comment (clemblanco) [reply CHANGELOG.md:10]: So far the `CHANGELOG.md` is the same for all versions and include the changes from all versions.

If we would prefer them to be version specifics, we would need to tackle this through another PR.

File: README.md [hunk 4]
```diff
@@ -13,16 +13,22 @@ This can be useful when wanting to store emails sent for archive purposes.
 
 You can install the package via composer:
 
-For Laravel 5.x and 6.x
+For Laravel 9.x
 
 ```bash
-composer require pod-point/laravel-mail-export:^0.1
+composer require pod-point/laravel-mail-export
```
[the remark this answers is not in the document]
Comment (clemblanco) [reply README.md:19]: Yeah same, `README.md` is the same too.

If we would prefer them to be version specifics, we would need to tackle this through another PR.

So far this makes it easier to maintain as we just copy/paste the files across, also hints the devs they should upgrade.

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (oaklees) [reply .gitignore]: lol - doesn't make it right 😉

**Pull request merged**